packages/cli/README.md

```diff
@@ -1,19 +1,44 @@
 # The Redwood Command Line Interface
 
-The Redwood CLI comes with RedwoodJS (which means no extra software to install!).
+The Redwood CLI comes with RedwoodJS (which means no extra software to install!). It is made with [yargs](https://yargs.js.org/).
 
 ## Usage
 
 The [`yarn`](https://classic.yarnpkg.com/en/docs/install) package is required to use the Redwood CLI.
 
-Be sure to prefix all Redwood CLI commands with `yarn `. For example, `yarn redwood new`.
+Be sure to prefix all Redwood CLI commands with `yarn`. For example, `yarn redwood new`.
```

Comment:
`yarn rw new` doesn't exist anymore. We should probably remove it.

Reply:
Done.